gst/isomp4/qtdemux.c

```diff
@@ -4191,22 +4111,11 @@ gst_qtdemux_decorate_and_push_buffer (GstQTDemux * qtdemux,
 {
   GstFlowReturn ret = GST_FLOW_OK;
 
+#ifdef HAVE_FLUC
   if (G_UNLIKELY (stream->encrypted)) {
-    GstBuffer *dec_buf = NULL;
-
-    GST_DEBUG_OBJECT (qtdemux, "decrypting stream index = %d track index %d",
-        stream->senc_sample_index, stream->track_id);
-    /* send the decrypt signal */
-    g_signal_emit (qtdemux, gst_qtdemux_signals[SIGNAL_DECRYPT], 0,
-        stream->track_id, buf, stream->senc_sample_index++, &dec_buf);
-
-    gst_buffer_unref (buf);
-    if (dec_buf) {
-      buf = dec_buf;
-    } else {
-      goto exit;
-    }
+    buf = fluc_drm_buffer_new_from_cenc (buf, stream->cenc_context);
```

Comment:
Who is unreffing the buf in param ?

Reply:
we're loosing the reference here, because buf param is being put to gststructure